src/redux/loaderSlice.tsx

```diff
@@ -0,0 +1,31 @@
+import { PayloadAction, createSlice } from "@reduxjs/toolkit"
+
+export interface ILoaderState {
+    isLoading: boolean,
+
+}
+const initialState: ILoaderState = {
+    isLoading: false,
```

Comment:
add counter to the state to manage multiple request at the same time, increase on request decrease on response and isLoading value = counter === 0